src/components/table.js

```diff
@@ -113,6 +157,50 @@ const activityDetailColumns = (object)=>[
             }
         }
     },
+    ...object=='repo'?
+        [{
+            title: t('大屏'),
+            dataIndex: 'name',
+            align:'center',
+            width: '20%',
+            //根据行号判断，前300的项目可点击大屏
+            render: function (text, row, index) {
+                //console.log('andy',row);
+                let [org_name,repo_name]=text.split("/");
+                //console.log(year);
+                //console.log(month);
+                let myyear=year;
+                let mymonth=month;
+                switch (mymonth) {
+                    case 0:
+                    {
+                        myyear--;
+                        mymonth='12';
+                        break;
+                    }
+                    case 10:
+                    {
+                        mymonth='10';
+                        break;
+                    }
+                    case 11:
+                    {
+                        mymonth = '11';
+                        break;
+                    }
+                    default:
+                        mymonth='0'+month;
+                        break
+                }
+                //console.log(myyear);
+                //console.log(mymonth);
+                let param='{\"org_name\":\"'+org_name+'\",\"repo_name\":\"'+repo_name+'\",\"t_month\":\"'+myyear+'-'+mymonth+'-01\"}';
+                //console.log(param);
+                param=window.btoa(param);
+                //console.log(param);
+                return <a href={"http://dataease.nzcer.cn/link/LeMILNSw?attachParams="+param} target="_blank">{repo_name}大屏看板</a>
+            }
```

Comment:
3. this code snippet repeats several times in this file, better extract it to a function then call the function any where~